docs/dev-guide/integration/rest-api/consumer/applicability.rst

```diff
@@ -87,6 +85,13 @@ and postponed until the current task is completed.
   }
  }
 
+:group_call_report:`_` ::
+
+ {
+     "_href": "/pulp/api/v2/task_groups/16412fcb-06fa-4caa-818b-b103e2a9bf44/state_summary/",
```

Comment:
@mhrivnak  I would love to add a link to the root of the resource, but it would look ugly. We currently use the reverse() method provided by django formulate the URL for the task_group_summary view. Since we don't have a view that represents the task_group resource, we can't do a reverse look up for it. I could just manually remove the state_summary from the URL we have now. Thoughts?